### PR TITLE
cards-components

### PR DESCRIPTION
**Pull request by anshtripathi01**
Please review my cards components
Cards

### CODE REVIEW

File: Cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    height: 55rem;
+    box-shadow: 0px 1px 4px 0px rgba(0, 0, 0, 0.3);
+    flex-wrap: wrap;
+}
+.card{
+  
+    box-shadow: 0px 0px 0px 1px rgba(0,0,0, 0.1);
+    margin-top:7px;
+    width: 180px;
+    height: 250px;
```
Comment (shashikantmaurya0007) [inline Cards/cards.css:23]: use relative value

**Review by shashikantmaurya0007 — COMMENTED** [review]
those are the few changes i have suggested rest looks good .you can merge this

File: Cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    width: 50rem;
+    height: 55rem;
+    box-shadow: 0px 1px 4px 0px rgba(0, 0, 0, 0.3);
+    flex-wrap: wrap;
+}
+.card{
+  
+    box-shadow: 0px 0px 0px 1px rgba(0,0,0, 0.1);
+    margin-top:7px;
+    width: 180px;
```
Comment (zabihshaik) [inline Cards/cards.css:22]: Better to use rems instead of px

File: Cards/cards.css [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+   overflow: hidden;
+   border-radius: 5px;
+  
+}
+.card-button{
+    box-shadow: 0px 0px 8px 0px rgba(0,0,0, 0.1);
+    border-radius: 5px;
+    padding: 7px;
+  border: none;
+
```
Comment (zabihshaik) [inline Cards/cards.css:36]: Better to use Prettier for formatting

File: Cards/cards.html [hunk 4]
```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ApnaUI|CardsUI</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```
Comment (zabihshaik) [inline Cards/cards.html:10]: Better to use Indentation

**Review by zabihshaik — COMMENTED** [review]
Everything looks Good. Just few suggestions:
1. Better to use rems instead of px
2. Better to use Prettier for formatting Code and Use Indentation
3. Add meta tag with your name

File: Cards/cards.html [hunk 5]
```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (zabihshaik) [inline Cards/cards.html:7]: Add meta tag with your name

**Pull request merged**